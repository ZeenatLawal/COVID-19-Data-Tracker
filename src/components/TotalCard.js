import React from 'react';
import Card from 'react-bootstrap/Card';

const TotalCard = (prop) => {
  const { name, total } = prop;

  return (
    <Card className="border-0">
      <Card.Title>{name}</Card.Title>
      <Card.Body>
        <Card.Text className="d-flex flex-column">
          <span>{total.toLocaleString()}</span>
          Confirmed Cases
        </Card.Text>
      </Card.Body>
    </Card>
  );
};

export default TotalCard;
