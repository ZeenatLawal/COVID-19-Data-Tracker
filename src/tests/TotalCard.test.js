import renderer from 'react-test-renderer';
import TotalCard from '../components/TotalCard';

describe('Filter component', () => {
  test('Snapshot test', () => {
    const totalCard = renderer.create(
      <TotalCard name="AFRICA" total="200987" />,
    )
      .toJSON();
    expect(totalCard).toMatchSnapshot();
  });
});
