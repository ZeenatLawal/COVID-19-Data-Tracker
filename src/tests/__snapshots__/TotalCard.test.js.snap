// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filter component Snapshot test 1`] = `
<div
  className="border-0 card"
>
  <div
    className="card-title h5"
  >
    AFRICA
  </div>
  <div
    className="card-body"
  >
    <p
      className="d-flex flex-column card-text"
    >
      <span>
        200987
      </span>
      Confirmed Cases
    </p>
  </div>
</div>
`;
